ruby-challenges.rb

```diff
@@ -19,12 +30,36 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sumOfArray arr
+    totalOfArray = 0
+    arr.map do | k |
+        totalOfArray += k
+    end
+    totalOfArray
+end
+
+p sumOfArray nums_array1
+p sumOfArray nums_array2
```

Comment:
Great!  Ruby also has a built-in method called .sum that does a lot of this work for you.